yktracec/.clangd

```diff
@@ -1,2 +1,2 @@
 CompileFlags:
-    CompilationDatabase: ../target/compile_commands/ykllvmwrap/
+    CompilationDatabase: ../target/debug/clangd/ykllvmwrap/
```

Comment:
While we are here, may aswell update this last remaining `ykllvmwrap`.

It should be `yktracec`, but there is another place somewhere in the build system still referencing `ykllvmwrap` as the LSP is actually working for `yktracec` :P

Reply:
Ooo, good spot! Fixed in 817d9e2.